src/number_theory/legendre_symbol/quadratic_char.lean

```diff
@@ -365,10 +386,45 @@ section quad_char_mod_p
 
 /-- Define the nontrivial quadratic character on `zmod 4`, `χ₄`.
 It corresponds to the extension `ℚ(√-1)/ℚ`. -/
-
 @[simps] def χ₄ : (zmod 4) →*₀ ℤ :=
 { to_fun := (![0,1,0,-1] : (zmod 4 → ℤ)),
-  map_zero' := rfl, map_one' := rfl, map_mul' := by dec_trivial }
+  map_zero' := rfl, map_one' := rfl, map_mul' := dec_trivial }
+
+/-- An explicit description of `χ₄` on integers / naturals -/
+lemma χ₄_of_mod_four_int (n : ℤ) : χ₄ n = if n % 2 = 0 then 0 else if n % 4 = 1 then 1 else -1 :=
```

Comment:
`of` suggests that the argument is an integer mod 4. The first thing that popped into my mind is:
```suggestion
lemma χ₄_int_eq_if_mod4 (n : ℤ) : χ₄ n = if n % 2 = 0 then 0 else if n % 4 = 1 then 1 else -1 :=
```
But I guess there is room for improvement.

Reply:
`χ₄_int_eq_if_mod_four` seems to be more consistent with what is already there and taks about residues mod 4.